ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -1461,6 +1468,17 @@ public void onRemoved() {
             mRootView.removeCallbacks(mReHideTextAction);
         }
 
+        public void setCloseIconDrawable(Drawable closeIcon) {
+            mCloseIcon = closeIcon;
```

Comment:
Can you apply this `drawable` to the `ImageView` here?

Reply:
removed this method in cleanup